client/src/modules/patients/templates/action.cell.html

```diff
@@ -32,5 +32,10 @@
         <span class="fa fa-money"></span> <span translate>PATIENT_REGISTRY.PAYMENTS</span>
       </a>
     </li>
+    <li>
+      <a data-method="voucher" ui-sref="vouchers({ filters : [{ key : 'entity_uuid', value : row.entity.debtor_uuid, displayValue: row.entity.display_name }]})" href>
```

Comment:
This is a nice API :+1:. I think for anything more than one filter it would be good to have this either pre-calculated for each row or generated by a method, although that may come with significantly more cost.